examples/animation/locomotion.html

```diff
@@ -0,0 +1,431 @@
+                            "conditions": [
+                                {
+                                    "parameterName": "speed",
+                                    "predicate": "GREATER_THAN",
+                                    "value": 0
+                                }
+                            ]
+                        },
+                        {
+                            "from": "ANY",
+                            "to": "Jump",
+                            "time": 0.1,
+                            "priority": 0,
+                            "conditions": [
+                                {
+                                    "parameterName": "jump",
+                                    "predicate": "EQUAL_TO",
+                                    "value": true
+                                }
+                            ]
+                        },
+                        {
+                            "from": "Jump",
+                            "to": "Idle",
+                            "time": 0.2,
+                            "priority": 0,
+                            "exitTime": 0.8
+                        },
+                        {
+                            "from": "Jump",
+                            "to": "Walk",
+                            "time": 0.2,
+                            "priority": 0,
+                            "exitTime": 0.8
+                        },
+                        {
+                            "from": "Walk",
+                            "to": "Idle",
+                            "time": 0.1,
+                            "priority": 0,
+                            "conditions": [
+                                {
+                                    "parameterName": "speed",
+                                    "predicate": "LESS_THAN_EQUAL_TO",
+                                    "value": 0
+                                }
+                            ]
+                        },
+                        {
+                            "from": "Walk",
+                            "to": "Jog",
+                            "time": 0.1,
+                            "priority": 0,
+                            "conditions": [
+                                {
+                                    "parameterName": "speed",
+                                    "predicate": "GREATER_THAN",
+                                    "value": 1
+                                }
+                            ]
+                        },
+                        {
+                            "from": "Jog",
+                            "to": "Walk",
+                            "time": 0.1,
+                            "priority": 0,
+                            "conditions": [
+                                {
+                                    "parameterName": "speed",
+                                    "predicate": "LESS_THAN",
```

Comment:
you should probably use pc.ANIM_LESS_THAN and similar in these instead of strings, as that's the API, similar to how you use pc.ANIM_PARAMETER_INTEGER few lines down.